### PR TITLE
fix(src): add new improvements and changes to design

### PR DESCRIPTION
**Pull request by hanakoori01**
### What does this PR do?

- Fixes and improvements to the design 
- Add content to the page

### Steps to test 
1. Run `yarn build`
2. Run `yarn start`
3. Go to `localhost:3000`
4. Go to test the page

### CODE REVIEW

File: docs/docs/about/team.md [hunk 1]
```diff
@@ -6,3 +6,13 @@ description: Meet the team of scientist and developers and engineers at Tikuna
...
+
+Tikuna is a proof-of-concept P2P network security monitoring system for the Ethereum blockchain. Using Machine Learning techniques, it will extract security and performance insights for detecting relevant incidents and improving visibility to the security state of the P2P network. This proposed research project has been approved and supported by the Ethereum Foundation with an award of USD 102,200 to start the development process.
+
+Tikuna team has approached this 2022 opening in the area of P2P Network to the Ethereum Foundation Academic Grants. We are the only team in Latin America that received approval and funding from ETH Foundation. This is the [official announcement](https://blog.ethereum.org/2022/07/29/academic-grants-grantee-announce) published by them.
+
+## The team behind Tikuna
+
+![edenia logo](/img/second-card.webp)
```
Comment (xavier506) [inline docs/docs/about/team.md:18]: Edenia Logo?

File: docs/src/pages/index.js [hunk 2]
```diff
@@ -37,10 +37,7 @@ const Home = () => {
   const { siteConfig } = useDocusaurusContext();
 
   return (
-    <Layout
-      title={`Hello from ${siteConfig.title}`}
-      description="Description will go into a meta tag in <head />"
-    >
+    <Layout title={`Hello from ${siteConfig.title}`} description="">
```
Comment (xavier506) [inline docs/src/pages/index.js:40]: Hello from?

Comment (hanakoori01) [reply docs/src/pages/index.js:40]: What would you like to put there?

**Review by tetogomez — APPROVED** [review]
LGTM

**Pull request merged**